src/components/ModelizerTextView.vue

```diff
@@ -41,9 +42,12 @@ import {
   ref,
   watch,
 } from 'vue';
-import { getProjectFiles, readProjectFile } from 'src/composables/Project';
+import { getProjectFiles, readProjectFile, writeProjectFile } from 'src/composables/Project';
 import FileEvent from 'src/composables/events/FileEvent';
 import GitEvent from 'src/composables/events/GitEvent';
+import PluginEvent from 'src/composables/events/PluginEvent';
+import { getPlugins } from 'src/composables/PluginManager';
+import { FileInformation } from 'leto-modelizer-plugin-core';
 
 const props = defineProps({
   viewType: {
```

Comment:
to delete if `viewType` prop not passed to `MonacoEditor.vue`

Reply:
Done